config/README.md

```diff
@@ -84,8 +84,7 @@ general:
   min_epochs: 1 # for train only
   raw_data_dir: data
   raw_data_csv: tests/tiling/tiling_segmentation_binary_ci.csv
-  sample_data_dir: data # where the hdf5 will be saved
-  state_dict_path:
```

Comment:
Why removing `state_dict_path` ? This is not relate to hdf5, its for the pth file if I remember.

Reply:
removing from README simply because this param no longers exists in our gdl_config_template: https://github.com/NRCan/geo-deep-learning/blob/develop/config/gdl_config_template.yaml#L35